layouts/partials/flex/head.html

```diff
@@ -14,7 +14,7 @@
 <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
 
 {{ if (fileExists "data/css.toml") -}}
-      <link href="{{ .Site.BaseURL }}css/{{.Site.Data.css.file | relURL}}" rel="stylesheet">
+      <link href="{{ .Site.BaseURL }}/css{{.Site.Data.css.file | relURL}}" rel="stylesheet">
```

Comment:
This may be better option:
```suggestion
      <link href="{{ .Site.BaseURL | replURL }}css{{.Site.Data.css.file | relURL}}" rel="stylesheet">
```